engine/api/src/main/java/io/deephaven/engine/table/Table.java

```diff
@@ -363,10 +373,33 @@ enum DirectionOptions {
         RTL
     }
 
+    /**
+     * Produce a new table with data bars generated at the specified column
```

Comment:
The first and only necessary param is column and that dictates which column will get the data bar format. It doesn't always mean the data bars will use the values in that column, that is defined by `value_column`. But it will be generated in the column passed in by the user which is what I meant by "specified column".